src/index.tsx

```diff
@@ -1465,6 +1522,12 @@ class App extends React.Component<{}, AppState> {
           </div>
           {someElementIsSelected() && (
             <>
+              <h4>Element's options</h4>
+              <div className="panelColumn">
+                {elementSelectedOptions.length > 0
+                  ? elementSelectedOptions
+                  : "No options available"}
```

Comment:
I think is better hide the "Element's options" then show the "No options available". Only show if there is a text element selected

Reply:
@paulomenezes done